src/test/java/stacks/StacksIT.java

```diff
@@ -90,7 +90,8 @@ public void hasCorrectStackHeadersIT() {
     @Test
     public void hasCorrectNumStackRowsIT() {
         int expectedNumberOfStacks = 5;
-        int actualNumberOfStacks = getNumTableRow("stack-table-body");
+        int actualNumberOfStacks = driver.findElements(By.xpath("//*[@id='stack-table-body']//tr[not(contains(@class, 'digest-notification'))]")).size();
```

Comment:
xpaths look like a mess lol lets try to not use them if possible. Does something like `driver.findElements(By.cssSelector("#stack-table-body tr:not(.digest-notification)"));` work?

Reply:
if that works try to substitute the rest of the xpaths below too